### PR TITLE
added model, index view, & controller for Parks, without user auth

### PR DESCRIPTION
**Pull request by leonrusso**

### CODE REVIEW

File: spec/features/parks/01_index_spec.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    # user = FactoryGirl.create(:user)
+
+  end
+
+  scenario "visitor signs in and views parks on index page" do
+
+    parks = FactoryGirl.create_list(:park, 2)
+    visit root_path
+
+    # sign_in_as(user)
```
Comment (davidrf) [inline spec/features/parks/01_index_spec.rb]: Delete this

File: spec/features/parks/01_index_spec.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+# So that I can pick items to review
+
+
+# TO DO : As an authenticated user
+
+feature "users can view parks" do
+  before do
+    # user = FactoryGirl.create(:user)
+
+  end
```
Comment (davidrf) [inline spec/features/parks/01_index_spec.rb]: Delete this `before` block

File: spec/features/parks/01_index_spec.rb (hunk 1 above)
Comment (leonrusso) [inline spec/features/parks/01_index_spec.rb]: what's this specifically?
we deleted the before block

Comment (davidrf) [inline spec/features/parks/01_index_spec.rb]: Line 21, it is commented out

Comment (leonrusso) [inline spec/features/parks/01_index_spec.rb]: it's part of our todo, once we have users merged, other pair is working on it now.
then we'll uncomment it and user auth. will be added to the test

Comment (davidrf) [inline spec/features/parks/01_index_spec.rb]: You don't need this for the test to pass though, right?? Unauthenticated users should probably be able to see the index for your parks. Delete it for now, and add it back in later if you want.

Comment (leonrusso) [inline spec/features/parks/01_index_spec.rb]: the user story suggested that we need the user to be authenticated to see the parks, but we'll take it out

**Pull request merged**